src/codegen/subexpression_elimination/common_subexpression_tracker.rs

```diff
@@ -34,25 +34,22 @@ bitflags! {
 }
```

Comment:
Is `Color` used anywhere?

Reply:
That is a remainder from my old code and should be removed. Thanks for catching this.